server/scripts/migrate.py

```diff
@@ -25,13 +25,16 @@
 COPY = False
 TEST_MODE = False
 
-DATA_PATH = './data/'
-RAW_DATA_PATH = './data/raw/'
-PROCESSED_DATA_PATH = './data/processed/'
-TEST_DATA_PATH = './data/test/'
-GIF_DATA_PATH = './data/gif/'
-URL_PREFIX = '/'
-ERROR_PATH = './data/error/'
+if len(sys.argv) != 8:
```

Comment:
Don't make everything required. All arguments should be optional as of now. See this example in paddle repo
https://github.com/PaddlePaddle/PaddleOCR/blob/0275354c934ab81e530b525288e85c184bc02f89/paddleocr.py#L129

Other values like `COPY` `TEST_MODE` `NUM_PICS` should be added to arg list as well.

Also line 43 - line 62 need to be changed to create folder for the default values if they don't exist. Additional checks need to be added for path provided by user. For user-input path, only existence need to be checked and don't create the folder for them.